### PR TITLE
Feature/saml2

### PR DESCRIPTION
**Pull request by ezturner**

### CODE REVIEW

File: docker/settings.py [hunk 1]
```diff
@@ -171,3 +171,14 @@
 SUPPORTTOOLS_PARENT_APP_URL = "/"
 
 REMOTE_USER_FORMAT = "uwnetid"
+
+MOCK_SAML_ATTRIBUTES = {
+    'uwnetid': ['javerage'],
+    'affiliations': ['student', 'member'],
+    'eppn': ['javerage@washington.edu'],
+    'scopedAffiliations': ['student@washington.edu', 'member@washington.edu'],
+    'isMemberOf': ['u_test_group', 'u_test_another_group'],
```
Comment (fanglinfang) [inline docker/settings.py]: It'd be better that this 'affiliations' mock data matches the mock PWS data
"EduPersonAffiliations":["member","student","alum","staff","employee"],

'isMemberOf':  ['u_astratst_myuw_test-support-admin']

File: travis-ci/settings.py [hunk 2]
```diff
@@ -138,3 +138,14 @@
 COMPRESS_ROOT = "compress_root"
 RESTCLIENTS_TEST_MEMCACHED = True
 RESTCLIENTS_MEMCACHED_SERVERS = ('localhost:11211', )
+
+MOCK_SAML_ATTRIBUTES = {
+    'uwnetid': ['javerage'],
+    'affiliations': ['student', 'member'],
```
Comment (fanglinfang) [inline travis-ci/settings.py]: Same as above

**Comment by fanglinfang** [comment]
How about bill, which is another user having override permission?

Are 'affiliations' and 'scopedAffiliations' come by default? 
As they will be sticky to the original user, is it possible to turn them off?

**Comment by ezturner** [comment]
It would actually appear that as long as you're using the Django development server you can still set REMOTE_USER just fine.

**Comment by ezturner** [comment]
We would encounter some issues if we were loading in data from the SAML assertions (say group membership) but we are not at the moment.

**Pull request merged**